inc/init.php

```diff
@@ -229,6 +229,12 @@
 
 require_once __DIR__.'/src/bootstrap.php';
 
+MyBB\app()->instance('config', new \Illuminate\Config\Repository());
```

Comment:
This may make more sense inside `inc/src/bootstrap.php`, or a service provider.

Reply:
I was thinking about introducing a proper service provider in 1.10 when config values would be moved & stored in the standardized `return []` format - in 1.9 additional config can be added to `$config` (which is why it's eventually unprefixed).

Reply:
It may make sense to introduce the service provider in 1.9 anyway - that way we can simply extend it in 1.10 and have the service provider available for plugin developers and such to start using sooner.